Library/Formula/git-fresh.rb

```diff
@@ -0,0 +1,16 @@
+class GitFresh < Formula
+  desc "Utility to keep git repos fresh"
+  homepage "https://github.com/imsky/git-fresh"
+  url "https://github.com/imsky/git-fresh/archive/v1.2.2.tar.gz"
+  sha256 "9c1a5026fb09ede9ee401c21321c491db2d6049a2f1d5d4a1cbdfd57f04d2e19"
+
+  def install
+    system "./install", prefix
+    bin.install "git-fresh"
```

Comment:
We prefer to use upstream install scripts instead of doing things by ourselves; so it’s up to you: either we keep `bin.mkpath` until the next version or you bump the version and `system "./install", bin` will work as is.